console/Cargo.toml

```diff
@@ -38,15 +38,17 @@ optional = true
 
 [features]
 default = [
-  "parallel",
```

Comment:
Re: for most of these - I'm not sure that it's desirable to remove parallel for the default feature set. A large number of the consumers will be nodes in the network and it's desirable to have this in the default feature set for them - because they too will have to experience in some cases not being able to change dependencies of dependencies and removing this default for them would likely negatively impact overall network health.

Reply:
In our discussion you explained how the Parallel feature wasn't working within most browsers. For the team's knowledge, can you elucidate what you were running into there? I think pointing out what the issues were will help us find a solution to this.

A few ideas on the above:
1. Have we yet looked into possibly using webworker extensions to Rayon with something like https://github.com/GoogleChromeLabs/wasm-bindgen-rayon
2. Are we indeed having the problem of nested rust dependencies - or are we fortunate enough to be dealing with only top-level dependencies where we manually select for Parallel?

Reply:
It's still on the default for snarkVM crate so as long as most consumers use that directly it should work. I agree though if parallel is the right default, we should switch the feature flag to from `parallel` to `serial`. It doesn't make sense to have a `parallel` feature flag if it's the default.

It probably makes sense to move most of the `aleo-wasm` logic into here too. The `aleo` if I understand correctly is mostly a compile from aleo instructions to byte code so the wasm stuff there doesn't really have anything to do with it.

As for why it wasn't working, it's very much related to `rayon` and the https://github.com/GoogleChromeLabs/wasm-bindgen-rayon could be the solution. I haven't investigated it yet but I'm starting to. I'll report back what I find.

Either way it will add extra complexity. Not all web browser support web workers so it'll probably require the build to produce both a serial and parallel version along with the code to detect features and switch between them. I don't think it'll get us out of the problem of cleanly separating serial and parallel versions of this codebase.

Reply:
These are points to consider, will revisit this tomorrow.